OtripleS.Api.Infrastructure.Provision/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Get the latest master and merge to avoid additional charges and this file is not needed.

Reply:
Comment resolved.